app/screens/settings-screen/settings-screen.tsx

```diff
@@ -208,6 +217,14 @@ export const SettingsScreenJSX: ScreenType = (params: SettingsScreenProps) => {
       enabled: walletIsActive,
       greyed: !walletIsActive,
     },
+    {
+      category: translate("tippingLink.title"),
+      icon: "cash-outline",
+      id: "tippingLink",
+      action: () => copyToClipBoard(username),
+      enabled: walletIsActive,
+      greyed: !walletIsActive,
```

Comment:
this should also validate if the username has been set

![Screenshot_1631682778](https://user-images.githubusercontent.com/2079600/133374683-f7fc1f5b-7cc1-4ca4-8281-2ffc3235769b.png)